modules/guides/examples/javascript-udfs/drop-sqlpp-reference-function.sh

```diff
@@ -0,0 +1,3 @@
+curl -v http://localhost:8093/query/service \
+  -u Administrator:password \
+  -d 'statement=DROP FUNCTION GetBusinessDays'
```

Comment:
Best practice: semicolon at end of N1QL statement

```suggestion
  -d 'statement=DROP FUNCTION GetBusinessDays;'
```